packages/core/jsx-runtime/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "main": "dist/core.cjs.js",
+  "module": "dist/core.esm.js",
+  "browser": {
+    "./dist/core.cjs.js": "./dist/core.browser.cjs.js",
+    "./dist/core.esm.js": "./dist/core.browser.esm.js"
```

Comment:
it's weird that these are named `core`, yeah? I tried to make it `jsx-runtime` but preconstruct overrode it. In any case, it works. We have:

`@emotion/core/dist/core.esm.js`
`@emotion/core/jsx-runtime/dist/core.esm.js`

and so on. The imports seem to work in the `minimal-react` so 🤷‍♀️ maybe it doesn't matter.

Reply:
yes, it doesnt matter - i agree the naming convention used for those files is confusing, we have already agreed in the past on making that change but I'm not sure when it will land